lib/kubeclient/common.rb

```diff
@@ -482,5 +487,14 @@ def http_options(uri)
 
       options.merge(@socket_options)
     end
+
+    def default_credentials_token
+      scopes =  ['https://www.googleapis.com/auth/cloud-platform']
+      authorization = Google::Auth.get_application_default(scopes)
```

Comment:
`#access_token` is a method on the object returned by `#get_application_default`, not on the `#apply` method result (it's not chainable). I could use `#tap` if you don't want the interim local variable but I tend to find that harder to read.